homeassistant/components/nina/config_flow.py

```diff
@@ -150,7 +148,10 @@ async def async_step_user(
                     vol.Required(CONF_MESSAGE_SLOTS, default=5): vol.All(
                         int, vol.Range(min=1, max=20)
                     ),
-                    vol.Optional(CONF_HEADLINE_FILTER, default=""): cv.string,
+                    vol.Optional(
+                        CONF_HEADLINE_FILTER, default=NO_MATCH_REGEX
+                    ): cv.string,
+                    vol.Optional(CONF_AREA_FILTER, default=ALL_MATCH_REGEX): cv.string,
```

Comment:
These are advanced options, move them to a [collapsed by default section ](https://developers.home-assistant.io/docs/data_entry_flow_index?_highlight=data&_highlight=entry#grouping-of-input-fields)

The section's description should explain what the filters are for